patches/common/chromium/.patches.yaml

```diff
@@ -322,3 +322,8 @@ patches:
     https://chromium.googlesource.com/chromium/src/+/f47b361887a31cccf42a6e21a82bccf28372bdaa%5E%21
     In the long term we should investigate why it happened, and take a more
     formal fix. But for now I'm just make this assertion silently pass away.
+
+  owners: torycl
+  file: backport_e3a883075.patch
+  description: |
+    https://chromium.googlesource.com/chromium/src/+/e3a883075f699e67ab47d9991ac55b143017cfc3
```

Comment:
Please add a line saying the this change has landed in version `67.0.3380.0`.
It will make it easier to make a decision if it's still needed at the start of the next Chromium upgrade.

Reply:
Done

Reply:
Thank you for review